data/tlg1763/__cts__.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<ti:textgroup xmlns:ti="http://chs.harvard.edu/xmlns/cts" projid="greekLit:tlg1763"
+	urn="urn:cts:greekLit:tlg1763">
+	<ti:groupname xml:lang="eng">Tryphon Grammaticus</ti:groupname>
```

Comment:
@AlisonBabeu would you change groupname lang to lat here?

Reply:
Awesome. I wondered about the name. My first instinct was to put "lat," but then I saw that "Tiberius Rhetor" was marked as "eng," so I wondered if that's just the convention we use. (Since in an English conversation we'd also just use the Latinate names.) I can see either making sense.